app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  has_secure_password validations: false
+  validates_presence_of :email, :password, :full_name
+  validates_uniqueness_of :email
+
+end
```

Comment:
No, at the very end of the file. It isn't needed but is a good practice - http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline